dids/didjwk.go

```diff
@@ -74,8 +74,10 @@ func NewDIDJWK(opts ...NewDIDJWKOption) (BearerDID, error) {
 	return did, nil
 }
 
+type JWKResolver struct{}
+
 // Resolves the provided DID URI
-func ResolveDIDJWK(uri string) (ResolutionResult, error) {
+func (r JWKResolver) Resolve(uri string) (ResolutionResult, error) {
```

Comment:
Unrelated to this PR, and it was actually my suggestion originally 🤦‍♂️, but error types in Go are idiomatically named `Err<something>`, eg. `ErrResolution`

Reply:
ah nice ok cool @alecthomas i'll address that in the restructure PR